### PR TITLE
SAL-14  add handlers for lambda or http, for user comments

### LINKED ISSUE #5: Create basic go http server
Create a basic http server that has the routes api/v1/comments and api/v1/users routes with stubs for functions.

### PR DESCRIPTION
**Pull request by grevian**
Fixes #5

### CODE REVIEW

File: cmd/http/main.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	"github.com/go-chi/chi/v5"
+	"github.com/go-chi/chi/v5/middleware"
+
+	"github.com/salt-today/salttoday2/internal/lambdas/api"
+)
+
+func main() {
+	r := chi.NewRouter()
+	r.Use(middleware.Logger)
+	r.Get("/users/{user_id}", api.GetUserCommentsHTTPHandler)
```
Comment (DanHarasymiw) [inline cmd/http/main.go:15]: can we do api/v1/users? 
Just for separation from a potential /users webpage 🤷

File: internal/lambdas/api/get_user_comments.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/go-chi/chi/v5"
+	"github.com/samber/lo"
+
+	"github.com/salt-today/salttoday2/internal/store"
+)
+
+func GetUserCommentsLambdaHandler(ctx context.Context, request events.APIGatewayProxyRequest) (events.APIGatewayProxyResponse, error) {
```
Comment (DanHarasymiw) [inline internal/lambdas/api/get_user_comments.go:19]: I'm for just scrapping the lambda stuff

File: cmd/http/main.go (hunk 1 above)
Comment (DanHarasymiw) [reply cmd/http/main.go:15]: Oh wait, actually there might be some miscommunication/confusion around this. 
This is what we did in salttoday1 so by far does not need to be the concrete way of doing things:
We supported getting user's comments via `/api/v1/comments?user=xyz`
Our `/api/v1/users` endpoint was for display a list of users

Comment (grevian) [reply cmd/http/main.go:15]: I think in either case this url is a little weird, but I was going to try to make things fairly RESTY, so ideally the primary id is in the path, not the parameters

**Pull request merged**